packages/help-center/src/components/help-center-content.tsx

```diff
@@ -117,8 +117,8 @@ const HelpCenterContent: React.FC< { isRelative?: boolean; currentRoute?: string
 					<Route
 						path="/"
 						element={
-							initialRoute ? (
-								<Navigate to={ initialRoute } />
+							navigateToRoute ? (
+								<Navigate to={ navigateToRoute } />
```

Comment:
This is bad. You shouldn't make routes dynamic. You should dynamically go to a route. A route being one of pre-defined bunch.

Reply:
This was the suggested method for navigating on load. Doing this preserves the blank location history which makes navigating logic easier to manage. Using `navigate()` initializes the location history while making this dynamic preserves that state.